### DISCUSSION TITLE
Take into consideration previous record in `klog start --resume`?

### DISCUSSION THREAD

**Discussion by lx4r** [jotaen/klog#263] [Feature ideas & discussions]
Hi,
I'm using tags to categorize my entries. `klog start --resume` is very useful for me as I usually work on tasks from the same category for an extended period of time. After taking a break I can just do `klog start --resume` and a new entry with the same tag as the previous one is added to the log. However, when executing `klog start --resume` at the start of a day to continue working on a task from the same category as the one of the task I worked on yesterday, a new entry is added to the log but no tag is copied.

What do you think about making `klog start --resume` not only take into consideration the last entry in the current record but the last entry in general, even if it's in a different record?

Thanks in advance for the help :)

**Comment by jotaen** [comment]
Yeah, I see how that functionality could be useful. However, I think that this behaviour could be unexpected, because the usual semantics for klog is that records are fully self-contained and independent. There would also be some complexity around the definition of “the last entry in general”: is it the last entry that was added to the file, the last entry in the chronological sense, or the last entry to appear in the file (i.e. on the last line)?

I’ll keep this idea in mind, though, so thanks for bringing this up.

**Comment by nazarewk** [comment]
I was about to write about it, I'm just getting the last entry chronologically in my wrapper and was surprised it doesn't work this way in `klog start --resume`

**Comment by hovsater** [comment]
I actually ran into this today. I ran `klog start --resume` expecting it to copy the summary from whatever I was working on last yesterday, but it didn't. Instead it created an open-ended entry without a summary. That was somewhat unexpected.

**Comment by jotaen** [comment]
👉 https://github.com/jotaen/klog/issues/266
